internal/srv/app.go

```diff
@@ -96,3 +109,12 @@ func (s *Server) CreateApp(name string, chartPath string, namespace string) erro
 
 	return nil
 }
+
+func getHelmValues(content []byte) (chartutil.Values, error) {
+	values := chartutil.Values{}
+	if err := yaml.Unmarshal(content, &values); err != nil {
```

Comment:
Could you add an error log too, please? It makes finding where parsing what happened far simpler in the future.

Reply:
✅